CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_fivetran_log v1.3.0
+
+## New Model Alert 👶🏽
+[PR #98](https://github.com/fivetran/dbt_fivetran_log/pull/98) includes the following update:
+- We have added a new model, `fivetran_platform__user_audit` ([source](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_platform__user_audit)):
```

Comment:
Responding to your comment on the name of the new model. I actually really like your suggestion of `fivetran_platform__ audit_user_activity`. It matches with the naming convention of `fivetran_platform__audit_table` and accurately represents what one would find in the new table.

Reply:
Can we make this update throughout the PR.